portal/jam-embedded-api.md

```diff
@@ -14,46 +14,46 @@ Fiddler Jam Embedded is a JavaScript library that provides an API to integrate t
 
 Follow the steps below to include and use the Fiddler Jam Embedded library in your webpage:
 
-1. Provide the list of endpoints that will integrate it (you can use Fiddler Jam Embedded only with a pre-set list of URLs)
-
-    ```CSV
-    https://getfiddler.com, https://docs.telerik.com
-    ```
-
-1. Obtain and store your unique Fiddler Jam Embedded API key.
+1. Obtain your unique Fiddler Jam Embedded API key. [Learn more on how to generate your Fiddler Jam API key here...](#generating-api-key)
 
 1. Import the Fiddler Jam API script into your HTML page from the following CDN link: https://downloads.getfiddler.com/jam-embedded/fiddler-jam-embedded.js.
 
-1. Initialize the Fiddler Jam Embedded object with your Fiddler Jam API key. The Fiddler Jam Embedded object is attached to the [`window`](https://www.w3schools.com/jsref/obj_window.asp) object of the page as a property called `_fiddlerJamEmbedded`.
-
-    ```JavaScript
-    <html lang="en">
-        <head>
-            <meta charset='utf-8'>
-            <meta name='viewport' content='width=device-width,initial-scale=1'>
-        
-            <title>Fiddler Jam Embedded</title>
-            <script src="https://downloads.getfiddler.com/jam-embedded/fiddler-jam-embedded.js"></script>
-            <script>
-                const jam = window._fiddlerJamEmbedded;
-                jam.init({
-                    apiKey: '<your-unique-Fiddler-Jam-API-key-here>'
-                });
-                // .. custom implementation follows here
-            </script>
-        </head>
-    </html>        
+1. Initialize the Fiddler Jam Embedded object with your Fiddler Jam API key. The Fiddler Jam Embedded object is attached to the [`window`](https://www.w3schools.com/jsref/obj_window.asp) object of the page as a property called `_fiddlerJamEmbedded`. Use the `load` event of the Fiddler Jam Embedded object to initialize the Jam API and to add your custom implementation.
+
+    ```html
+        <html lang="en">
+            <head>
+                <meta charset='utf-8'>
+                <meta name='viewport' content='width=device-width,initial-scale=1'>
+            
+                <title>Fiddler Jam Embedded</title>
+
+                <script src="https://downloads.getfiddler.com/jam-embedded/fiddler-jam-embedded.js" id="jamEmbeddedScript" async></script>
+                <script>
+                    const jamEmbeddedScript = document.getElementById('jamEmbeddedScript');
+                    jamEmbeddedScript.crossOrigin = 'anonymous';  // set crossOrigin to enable CORS (and to be able to use the load event below)
```

Comment:
The correct name of the attribute is crossorigin: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin